cluster/cluster.go

```diff
@@ -299,17 +314,20 @@ func CreateDeploymentWithTenants(tenants []Tenant, storageClusterNum string, hos
 
 	for i := range tenants {
 		tenant := tenants[i]
+		envName := fmt.Sprintf("%s%s-env", deploymentPrefix, tenant.Name)
 		volumeMounts := []v1.VolumeMount{}
 		tenantContainer := v1.Container{
-			Name:            fmt.Sprintf("%s-minio-%s", tenant.Name, hostNum),
-			Image:           "minio/minio:RELEASE.2019-09-26T19-42-35Z",
-			ImagePullPolicy: "IfNotPresent",
+			Name: fmt.Sprintf("%s-minio-%s", tenant.Name, hostNum),
+			//Image:           "minio/minio:RELEASE.2019-10-02T21-19-38Z",
+			Image:           "y4m4/minio:dsync-2",
```

Comment:
You can use minio/minio:edge to get the merged fix.

Reply:
switched